### ISSUE TITLE
Edit Button Template button not working

### ISSUE THREAD

**Issue by Bohica72** [mProjectsCode/obsidian-meta-bind-plugin#434]
### Please fill out these Check-boxes

- [X] I checked for existing similar issues
- [X] I checked that the plugin is up to date
- [X] The issue persists with all other plugins and themes disabled

### Plugin Version

1.2.1

### This Issue Occurs on

- [X] Windows
- [ ] Linux
- [ ] macOS
- [ ] Android
- [ ] iOS

### Debug Info

```text
SYSTEM INFO:
	Obsidian version: v1.6.7
	Installer version: v1.6.7
	Operating system: Windows 11 Home 10.0.22635
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Base theme: adapt to system
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 1
	Plugins enabled: 1
		1: Meta Bind v1.2.1

RECOMMENDATIONS:
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

### Describe the Issue

Clicking the Edit button to edit a button template results in no action. Copy an delete buttons still work as expected. Behaviour persisted after all other plugins and custom css was uninstalled. 

This issue persisted on a fresh install of Obsidian with a clean vault (debug info provided for this vault)- where the only change to the default install was to add Meta Bind.

Strangely, everything was fine until a few days ago.

### Steps to Reproduce

obsidian settings>> meta bind >>edit button templates>> clicking on edit button.

### Expected Behavior

Button is expected to respond on click.

**Comment by christoffer31** [comment]
I have the same Issue on MacOS and iOS

**Comment by mProjectsCode** [comment]
Should be fixed in 1.2.2
